### PR TITLE
AKCORE-113: Update code to match KIP-932

### PR DESCRIPTION
**Pull request by AndrewJSchofield**
Update to KIP-932 v.70 definitions.

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: clients/src/main/resources/common/message/ShareGroupHeartbeatRequest.json
Comment (apoorvmittal10) [inline clients/src/main/resources/common/message/ShareGroupHeartbeatRequest.json:44]: Question: shall we fill this data as well in group metadata manager?

**Review by apoorvmittal10 — APPROVED** [review]
LGTM.

File: clients/src/main/resources/common/message/ShareGroupHeartbeatRequest.json
Comment (AndrewJSchofield) [reply clients/src/main/resources/common/message/ShareGroupHeartbeatRequest.json:44]: Not yet. The KIP has gone back on this again. Once we have approval, let's do a final refresh of all structures.

**Pull request merged**
